File: gpao-tex-core/ordre/src/main/java/com/gpro/consulting/ordre/dto/FicheEclatementValue.java
package com.gpro.consulting.ordre.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Calendar;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class FicheEclatementValue {
    Long id;
    List<PaquetValue> paquets;
    List<ChaineValue> chaines;
    Long ordreId;
    Calendar dateLancement;
    String observations;
    Long nombrePaquet;
    Long quantiteEclate;
    String codeConception;
    String clientAbreviation;
    Long produitId;
}

File: gpao-tex-ui/source/main/src/app/services/order/models/fiche-eclatement-value.ts
/* tslint:disable */
/* eslint-disable */
import { ChaineValue } from '../models/chaine-value';
import { PaquetValue } from '../models/paquet-value';
export interface FicheEclatementValue {
  chaines?: Array<ChaineValue>;
  clientAbreviation?: string;
  codeConception?: string;
  dateLancement?: string;
  id?: number;
  nombrePaquet?: number;
  observations?: string;
  ordreId?: number;
  paquets?: Array<PaquetValue>;
  produitId?: number;
  quantiteEclate?: number;
}
